### DISCUSSION TITLE
GCP Velero plugin - backup is showing New status with no progress

### DISCUSSION THREAD

**Discussion by venkatjaya** [vmware-tanzu/velero#3903] [Community support Q&A]
NAME                                STATUS      ERRORS   WARNINGS   CREATED                         EXPIRES   STORAGE LOCATION   SELECTOR
backup-1624636633142   Completed   0        0          2021-06-25 11:13:09 -0500 CDT   23h       default            <none>

**Comment by zubron** [comment]
Hi, I'm not sure what your question is - this looks to be the output from listing the backups and it is showing as "Completed".

If you having issues with backups we'll need the following: 

**The output of the following commands will help us better understand what's going on**:
(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)

- `kubectl logs deployment/velero -n velero`
- `velero backup describe <backupname>` or `kubectl get backup/<backupname> -n velero -o yaml`
- `velero backup logs <backupname>`
